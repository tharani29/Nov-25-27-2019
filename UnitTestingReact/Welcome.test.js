import React from 'react';
import { shallow } from 'enzyme';
import Welcome from './Welcome';

describe("Welcome", () => {
	it("should render properly", () => {
		const component = shallow(<Welcome/>);
		expect(component).toMatchSnapshot();
	});
	
	it("should contain header text", () => {
		const component = shallow(<Welcome/>);
		expect(component.find("h3").text()).toEqual("Welcome to React Testing");
	});
});