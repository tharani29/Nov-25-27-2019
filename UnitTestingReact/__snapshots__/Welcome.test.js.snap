// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render properly 1`] = `
<h3>
  Welcome to React Testing
</h3>
`;
